kazoo.go

```diff
@@ -117,7 +117,31 @@ func (kz *Kazoo) Brokers() (map[int32]string, error) {
 			return nil, err
 		}
 
-		result[int32(brokerID)] = fmt.Sprintf("%s:%d", brokerNode.Host, brokerNode.Port)
+		if len(brokerNode.Host) == 0 || brokerNode.Port == -1 {
+			type brokerEndpoints struct {
+                		Endpoints []string `json:"endpoints"`
+		        }
+			var brokerEP brokerEndpoints
+
+			if err := json.Unmarshal(value, &brokerEP); err != nil {
+                        	return nil, err
+        	        }
+
+			if len(brokerEP.Endpoints) == 0 {
+                        	return nil, err
```

Comment:
You cannot use `err` here; you'll need the create your own error value to return for this scenario.